meta-ostro-xt/recipes-swupd/swupd-client/swupd-client/efi-combo-trigger.service.new

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Updates the EFI combo application
+BindsTo=update-triggers.target
+
+[Install]
+WantedBy=update-triggers.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/efi_combo_updater
```

Comment:
Should this be ExecStart=/usr/bin/efi_combo_updater.sh? It's installed below with install -m 0744 ${WORKDIR}/efi_combo_updater.sh ${D}/usr/bin/

Are users expected to invoke this command? If not, then I wouldn't put it into the default shell search PATH.